### DISCUSSION TITLE
VFP Tables - Reading and Interpolating

### DISCUSSION THREAD

**Discussion by ray63** [equinor/resdata#867] [Q&A]
I note resinsight can read and plot VFP tables but couldn't find the method that can read these tables.  My ultimate objective is to perform a lookup in a VFP table - are there any utilities available to (1) read the VFP table; and (2) peform the lookup?

**Comment by berland** [comment]
You may have a look at https://equinor.github.io/subscript/scripts/vfp2csv.html from https://github.com/equinor/subscript

**Reply by ray63** [reply]
Thanks Berland - have any code been written to interpolate the VFP tables?  The vfp2csv  is a great start.

**Reply by berland** [reply]
No code exists for interpolation as far as I know, but certainly something similar to https://github.com/equinor/pyscal is relevant for VFP tables. A start would perhaps be to reimplement this vfp2csv as requested in https://github.com/equinor/ecl2df/issues/7
